lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosResult.cxx

```diff
@@ -104,11 +104,142 @@ String FunctionalChaosResult::__repr__() const
          << " metaModel=" << metaModel_;
 }
 
-String FunctionalChaosResult::__str__(const String & offset) const
+String FunctionalChaosResult::__str__(const String & /*offset*/) const
+{
+  return __repr_markdown__();
+}
+
+/* String formatter with blank space right padding */
+String FunctionalChaosResult::padString(const String & string) const
```

Comment:
Is there the proper place for this method and the following? Shouldn't they be moved to String.hxx as static methods?

Reply:
There is no String.hxx, but [OTtypes.hxx](https://github.com/openturns/openturns/blob/457622ecf5264389f3739316e9c8cdc92aaea083/lib/include/openturns/OTtypes.hxx#L46). How to proceed then?

Reply:
Fixed at 230c32b29